exercises/html-relative-links/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Home Page</title>
+  <style>
+    img {
+      width: 300px;
+    }
+  </style>
+</head>
+<body>
+  <ul>
+    <li><a href="index.html">Home Page</a></li>
+    <li><a href="about-me/about-me.html">About Me</a></li>
+    <li><a href="projects/projects.html">Projects</a></li>
+    <li><a href="contact-us/contact-us.html">Contact Us</a></li>
+  </ul>
+  <div>
+    Hello Homepage!
+  </div>
+  <img src="../html-relative-links/images/javascript.png" alt="Girl holding programming book">
```

Comment:
This path can be "images/javascript.png". You don't need to exit the directory here, the `images` directory is in the same space as this file